CHANGELOG.md

```diff
@@ -4,17 +4,37 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.20 - 2023-10-03
+
+### Added
+- Documentation for experimental option `--pass-args` [#3229](https://github.com/earthly/earthly/pull/3229).
```

Comment:
this isn't related to the changes between earthly v0.7.19 and v0.7.20; generally the changelog should only list items that are related to noticeable changes in the binary itself.

I would delete this line.